packages/blade/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,290 @@
+import { Link } from '~components/Link';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography';
+import { isReactNative } from '~utils';
+import { List, ListItem } from '~components/List';
+import { IconButton } from '~components/Button/IconButton';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
+import { Sandbox } from '~src/_helpers/storybook/Sandbox';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentName="Switch"
+      componentDescription="A switch component is used to quickly switch between two possible states. These are only used for binary actions that occur immediately after the user turn the switch on/off."
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13227%3A163026',
+        bankingTheme:
+          'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=11169%3A230354',
+      }}
+    >
+      <Title>Usage</Title>
+      <Sandbox showConsole>
+        {`
+        import { Switch } from '@razorpay/blade/components'
+        
+        function App(): JSX.Element {
+          return (
+            // Check console
+            <Switch onChange={(e) => console.log(e.isChecked)} accessibilityLabel="Toggle DarkMode" />
+          )
+        }
+
+        export default App;
+      `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/Tooltip',
+  component: TooltipComponent,
+  args: {
+    placement: 'bottom',
+    content: 'Amount reversed to customer bank account',
+    onOpenChange: ({ isOpen }) => {
+      console.log(isOpen);
+    },
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<TooltipProps>;
+
+const Center = ({ children }: { children: React.ReactNode }): React.ReactElement => {
+  return (
+    <Box width="100%" height="100%" display="flex" alignItems="center" justifyContent="center">
+      {children}
+    </Box>
+  );
+};
+
+const TooltipTemplate: ComponentStory<typeof TooltipComponent> = (args) => {
+  return (
+    <Center>
+      <TooltipComponent {...args}>
+        <Button>Hover over me</Button>
+      </TooltipComponent>
+    </Center>
+  );
+};
+
+export const Default = TooltipTemplate.bind({});
+Default.storyName = 'Default';
+
+const PlacementBox = React.forwardRef<
+  HTMLDivElement,
+  { children: React.ReactNode } & TooltipTriggerProps
+>(({ children, ...props }, ref) => {
+  return (
+    <Box
+      ref={ref}
+      tabIndex={0}
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      flex={isReactNative() ? undefined : 1}
+      width={isReactNative() ? '40%' : '100%'}
+      flexShrink={0}
+      padding="spacing.5"
+      backgroundColor="surface.background.level3.lowContrast"
+      {...props}
+    >
+      <Text contrast="low">{children}</Text>
+    </Box>
+  );
+});
+
```

Comment:
lets also add a dedicated story for showing a11y specific things and props

Reply:
🤔 Not sure if we need a dedicated story for that, all the stories are a11y complete, just open the screenreader and good to go. Consumers don't need to do anything to for a11y.